build/webpack.base.config.js

```diff
@@ -29,7 +29,6 @@ function getCssLoaders(config) {
         {
             loader: 'css-loader',
             options: {
-                minimize: isProd,
```

Comment:
This was removed, clients must now do minification through `postcss-loader`/`cssnano` or `optimize-cssnano-plugin ` - https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#100-2018-07-06